pkg/config/config.go

```diff
@@ -73,6 +73,12 @@ func (a *App) Init() error {
 		{a.Web.MainUsernameClaim == "", "no output web.mainUsernameClaim specified, using default: 'name'", func() {
 			a.Web.MainUsernameClaim = "name"
 		}},
+		{a.Web.TemplatesDir == "", "no web.templatesDir specified, using default: '/web/templates'", func() {
+			a.Web.TemplatesDir = "/web/templates"
+		}},
+		{a.Web.AssetsDir == "", "no web.assetsDir specified, using default: '/web/assets'", func() {
+			a.Web.AssetsDir = "/web/assets"
+		}},
```

Comment:
I don't think this is required since you use default values in viper flags setup

Reply:
Fixed